common/gsutil.py

```diff
@@ -19,29 +19,29 @@
 logger = logs.Logger('gsutil')
 
 
-def gsutil_command(arguments, *args, parallel=False, **kwargs):
+def command(arguments, *args, parallel=False, **kwargs):
     """Executes a gsutil command with |arguments| and returns the result."""
-    command = ['gsutil']
+    command_list = ['gsutil']
```

Comment:
I think we should leave the name of this variable as command as well to be consistent with the rest of the code.
THis should be fine once the function is renamed.

Reply:
Right. But here I change it because I want to rename the function to `command()`.
Another thing is, `command_list` is actually a Python list, not a string. This renaming indicates its type.

Reply:
`command` needs to be deleted anyway. 
As for the type, I don't think that's important, it's more important to me that this code is consistent with other code such as `gcloud.py`+`new_process.py` then include type info that's clear from reading the code anyway.